HW4/Task2.py

```diff
@@ -0,0 +1,17 @@
+print("Homework 4. Task 2. Make list of simple multipliers", 
+        "of inserted number.")
+print()
```

Comment:
Писать условия таким образом, совершенно необязательно, но чтобы не делать еще один пустой `print`, можно сделать так.
```suggestion
print("Homework 4. Task 2. Make list of simple multipliers", 
        "of inserted number.", end='\n\n')
```